src/pynwb/retinotopy.py

```diff
@@ -1,145 +1,149 @@
-                     {'name': 'focal_depth_image', 'child': True},
-                     {'name': 'vasculature_image', 'child': True},
-                     'axis_descriptions')
-
-    @docval({'name': 'sign_map', 'type': RetinotopyMap,
-             'doc': 'Sine of the angle between the direction of the gradient in axis_1 and axis_2.'},
-            {'name': 'axis_1_phase_map', 'type': AxisMap,
-             'doc': 'Phase response to stimulus on the first measured axis.'},
-            {'name': 'axis_1_power_map', 'type': AxisMap,
-             'doc': 'Power response on the first measured axis. Response is scaled so 0.0 is no power in '
-                    'the response and 1.0 is maximum relative power.'},
-            {'name': 'axis_2_phase_map', 'type': AxisMap,
-             'doc': 'Phase response to stimulus on the second measured axis.'},
-            {'name': 'axis_2_power_map', 'type': AxisMap,
-             'doc': 'Power response on the second measured axis. Response is scaled so 0.0 is no '
-                     'power in the response and 1.0 is maximum relative power.'},
-            {'name': 'axis_descriptions', 'type': Iterable, 'shape': (2, ),
-             'doc': 'Two-element array describing the contents of the two response axis fields. '
-                    'Description should be something like ["altitude", "azimuth"] or ["radius", "theta"].'},
-            {'name': 'focal_depth_image', 'type': FocalDepthImage,
-             'doc': 'Gray-scale image taken with same settings/parameters (e.g., focal depth, wavelength) '
-                    'as data collection. Array format: [rows][columns].'},
-            {'name': 'vasculature_image', 'type': RetinotopyImage,
-             'doc': 'Gray-scale anatomical image of cortical surface. Array structure: [rows][columns].'},
-            {'name': 'name', 'type': str, 'doc': 'the name of this container', 'default': 'ImagingRetinotopy'})
-    def __init__(self, **kwargs):
-        axis_1_phase_map, axis_1_power_map, axis_2_phase_map, axis_2_power_map, axis_descriptions, \
-            focal_depth_image, sign_map, vasculature_image = popargs(
-                'axis_1_phase_map', 'axis_1_power_map', 'axis_2_phase_map', 'axis_2_power_map',
-                'axis_descriptions', 'focal_depth_image', 'sign_map', 'vasculature_image', kwargs)
-        super().__init__(**kwargs)
-        warnings.warn("The ImagingRetinotopy class currently cannot be written to or read from a file. "
-                      "This is a known bug and will be fixed in a future release of PyNWB.")
-        self.axis_1_phase_map = axis_1_phase_map
-        self.axis_1_power_map = axis_1_power_map
-        self.axis_2_phase_map = axis_2_phase_map
-        self.axis_2_power_map = axis_2_power_map
-        self.axis_descriptions = axis_descriptions
-        self.focal_depth_image = focal_depth_image
-        self.sign_map = sign_map
-        self.vasculature_image = vasculature_image
+raise ValueError(
```

Comment:
Should this be a DeprecationError instead?

Reply:
AFAIK this does not exist in the python standard library. DeprecationWarning does.

Reply:
From the built-in error types I think raising an `ImportError` would be more appropriate than a `ValueError` or the "catch-all" seems to be use a `RuntimeError`

Reply:
Good point. I changed it to a `RuntimeError`